scripts/gha/build_desktop.py

```diff
@@ -57,48 +57,6 @@ def append_line_to_file(path, line):
       file.write("\n" + line + "\n")
 
 
-def install_x86_support_libraries(gha_build=False):
```

Comment:
This function moved to utils.py so it can also be used by install_prereqs_desktop.py.

Reply:
is the utils.py included in this PR?

Reply:
It was modified here: https://github.com/firebase/firebase-cpp-sdk/pull/945